src/containers/orgs/components/OrgTitleSection.js

```diff
@@ -0,0 +1,193 @@
+/*
+ * @flow
+ */
+
+import React, { Component } from 'react';
+
+import styled from 'styled-components';
+import { Map } from 'immutable';
+import { OrganizationsApiActions } from 'lattice-sagas';
+import { Form } from 'lattice-fabricate';
+import { EditButton } from 'lattice-ui-kit';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { RequestStates } from 'redux-reqseq';
+import type { RequestSequence, RequestState } from 'redux-reqseq';
+
+import * as ReduxActions from '../../../core/redux/ReduxActions';
+
+const { UPDATE_ORG_TITLE } = OrganizationsApiActions;
+
+const dataSchema = {
+  properties: {
+    title: {
+      title: 'Edit organization\'s title',
+      type: 'string',
+    },
+  },
+  title: '',
+  type: 'object',
+};
+
+export const uiSchema = {
+  title: {
+    classNames: 'column-span-12',
+    'ui:widget': 'textarea',
+  },
+};
+
+const OrgTitleWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  margin: 20px 0 0 0;
+
+  > button {
+    margin-left: 16px;
+  }
+`;
+
+const OrgTitleHeader = styled.h1`
+  font-size: 28px;
+  font-weight: normal;
+  margin: 0;
+  padding: 0;
+`;
+
+const FormMinusMargin = styled(Form)`
+  flex: 1;
+  margin: -30px;
+`;
```

Comment:
Are these styles that should be factored into `lattice-fabricate`?

Reply:
`lattice-fabricate` has `StyledForm` which defines some important (and hard to understand) base formatting:

https://github.com/openlattice/lattice-fabricate/blob/develop/src/form/src/components/styled/StyledForm.js

I think the `margin: -30px` should definitely NOT happen, and `StyledForm` should be updated to reflect that. As for the `flex: 1`... meh... this one is harder to reason about.

Reply:
That makes sense about the margin. Ya these styles are definitely hard to undestand—Is the margin being set by RJSF? And ya, I think I agree about `flex: 1`.

Reply:
It's not a margin. It's actually a padding, set here:

https://github.com/openlattice/lattice-fabricate/blob/develop/src/form/src/components/styled/StyledForm.js#L27

The selector is complicated:

```
> div.form-group.field.field-object > :first-child {
  ...
  & > * {
    padding: 30px;
  }
  ...
}
```

It can also be applied to nested children. I only want the negative margin to apply at the highest level, and I haven't had time to untangle the CSS yet. The reason is polish.